src/content/3.6/code/agda/snippet03.agda

```diff
@@ -0,0 +1,4 @@
+record Monoid {m : Set} : Set where
```

Comment:
Perhaps `snippet03.agda` should just be a link to `snippet01.agda` so we're not repeating identical code...?

Reply:
Sounds like a good idea 👍